client/src/components/stadiumProfile/BioSection.tsx

```diff
@@ -33,22 +48,36 @@ const BioSection = () => {
           <StarIcon />
```

Comment:
These icons will be the rate of the stadium's reviews, not just statics ones.

Reply:
that will happen when creating protected routes